### PR TITLE
OSDOCS-11278: updates oc table and other fixes MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14, 4.15, 4.16, 4.17

Issue:
[OSDOCS-11278](https://issues.redhat.com/browse/OSDOCS-11278)

Link to docs preview:
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-cli-tools-introduction.html
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-cli-using-oc.html
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-oc-cli-commands-list.html
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-oc-cli-install.html
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-oc-config.html
https://81012--ocpdocs-pr.netlify.app/microshift/latest/microshift_cli_ref/microshift-usage-oc-kubectl.html

QE review:
- [x] QE has approved this change.

Additional information:
This is doc work which did not need an engineering SME.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
Placing a hold on merges until QE acks. OK to peer review.

File: microshift_cli_ref/microshift-usage-oc-kubectl.adoc [hunk 1]
```diff
@@ -47,16 +47,11 @@ Non-security API changes will involve, at minimum, two minor releases (4.1 to 4.
 |*X.Y+N* footnote:versionpolicyn[Where *N* is a number greater than or equal to 1.] (`oc` Client)
 
 |*X.Y* (Server)
-|image:redcircle-1.png[]
-|image:redcircle-3.png[]
+|Fully compatible.
+|The `oc` CLI tool might provide options and features that might not be compatible with the accessed server.
```
Comment (kowen-rh) [inline microshift_cli_ref/microshift-usage-oc-kubectl.adoc:51]: Using "might" twice here feels redundant. Perhaps:

```adoc
The `oc` CLI tool might provide options and features that are not compatible with the accessed server.
```

File: modules/cli-installing-cli-rpm.adoc [hunk 2]
```diff
@@ -79,7 +79,9 @@ endif::openshift-rosa[]
 # yum install openshift-clients
 ----
 
-After you install the CLI, it is available using the `oc` command:
+.Verification
+
+* After you install the the OpenShift CLI, it is available using the `oc` command:
```
Comment (kowen-rh) [inline modules/cli-installing-cli-rpm.adoc:84]: Typo: "the the"

File: microshift_cli_ref/microshift-oc-cli-install.adoc [hunk 3]
```diff
@@ -6,18 +6,13 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-To use the OpenShift CLI (`oc`) tool, you must download and install it separately from your {microshift-short} installation.
-
-[id="installing-the-openshift-cli"]
-== Installing the OpenShift CLI
-
-You can install the OpenShift CLI (`oc`) either by downloading the binary or by using Homebrew.
+To use the {oc-first} tool, you must download and install it separately from your {microshift-short} installation. You can install `oc` either by downloading the binary or by using Homebrew.
```
Comment (kowen-rh) [inline microshift_cli_ref/microshift-oc-cli-install.adoc:9]: "Either" is unnecessary here IMO.

File: modules/cli-installing-cli-brew.adoc [hunk 4]
```diff
@@ -15,9 +15,18 @@ For macOS, you can install the OpenShift CLI (`oc`) by using the link:https://br
...
+* Install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package by running the following command:
 +
 [source,terminal]
 ----
 $ brew install openshift-cli
 ----
+
+.Verification
+
+* After you install the the OpenShift CLI, it is available using the `oc` command:
```
Comment (kowen-rh) [inline modules/cli-installing-cli-brew.adoc:27]: Should "OpenShift CLI" be an attribute? It was replaced with one in earlier files.

File: modules/cli-installing-cli-rpm.adoc (hunk 2 above)
Comment (kowen-rh) [reply modules/cli-installing-cli-rpm.adoc:84]: Also, should "OpenShift CLI" be an attribute?

**Review by kowen-rh — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: microshift_cli_ref/microshift-usage-oc-kubectl.adoc [hunk 5]
```diff
@@ -6,36 +6,36 @@ include::_attributes/attributes-microshift.adoc[]
...
+If you installed an earlier version of `oc`, you might not be able use it to complete all of the commands in {microshift-short} {product-version}. If you want the latest features, you must download and install the latest version of `oc` that corresponds with your {microshift-short} version.
 ====
 
-Non-security API changes will involve, at minimum, two minor releases (4.1 to 4.2 to 4.3, for example) to allow older `oc` binaries to update. Using new capabilities might require newer `oc` binaries. A 4.3 server might have additional capabilities that a 4.2 `oc` binary cannot use and a 4.3 `oc` binary might have additional capabilities that are unsupported by a 4.2 server.
+Non-security API changes involve at least two minor releases, such as from 4.15 to 4.17, for example. But using new capabilities often requires the latest `oc` binary. A 4.17 server might have additional capabilities that a 4.12 `oc` binary cannot use and a 4.17 `oc` binary might have additional capabilities that are unsupported by a 4.13 server.
```
Comment (dhensel-rh) [inline microshift_cli_ref/microshift-usage-oc-kubectl.adoc:38]: Is the word `But` needed in the second sentence ?

File: modules/cli-installing-cli-brew.adoc [hunk 6]
```diff
@@ -15,9 +15,18 @@ For macOS, you can install the OpenShift CLI (`oc`) by using the link:https://br
 
 .Procedure
 
-* Run the following command to install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package:
+* Install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package by running the following command:
 +
 [source,terminal]
 ----
 $ brew install openshift-cli
```
Comment (dhensel-rh) [inline modules/cli-installing-cli-brew.adoc:22]: I know this is outside the scope of change, but `brew` does not have an `install` option like is presented here.

File: modules/cli-installing-cli-brew.adoc (hunk 4 above)
Comment (ShaunaDiaz) [reply modules/cli-installing-cli-brew.adoc:27]: Changing the wording of this line. It's kinda awkward anyway.

File: microshift_cli_ref/microshift-usage-oc-kubectl.adoc (hunk 5 above)
Comment (ShaunaDiaz) [reply microshift_cli_ref/microshift-usage-oc-kubectl.adoc:38]: Yeah....The way it was originally written, I think they meant to imply that `oc` can be used across two releases and still work with non-security API changes, BUT to ensure compatibility with everything else, you should just always update `oc` anyway. TBH that first sentence might not even be true anymore, so I just removed it.

**Pull request merged**